src/openforms/formio/typing/base.py

```diff
@@ -44,6 +44,9 @@ class OpenFormsConfig(TypedDict):
     translations: NotRequired[TranslationsDict]
     components: NotRequired[AddressValidationComponents]
     requireVerification: NotRequired[bool]
+    dataSrc: NotRequired[str]
```

Comment:
```suggestion
    dataSrc: NotRequired[Literal["manual", "variable", "referentielijst"]]
```
to complete with the correct literal values of course :)